common/src/main/java/whocraft/tardis_refined/client/screen/selections/ShellSelectionScreen.java

```diff
@@ -84,20 +84,19 @@ public void render(PoseStack poseStack, int i, int j, float f) {
         Lighting.setupForFlatItems();
         int k = (int) this.minecraft.getWindow().getGuiScale();
         RenderSystem.viewport((this.width - 320) / 2 * k, (this.height - 240) / 2 * k, 320 * k, 240 * k);
-        Matrix4f matrix4f = Matrix4f.createTranslateMatrix(-0.34F, 0.23F, 0.0F);
-        matrix4f.multiply(Matrix4f.perspective(Integer.MAX_VALUE, 1.3333334F, 9.0F, Integer.MAX_VALUE));
+        Matrix4f matrix4f = new Matrix4f().translate(-0.34F, 0.23F, 0.0F);
+        matrix4f.mul(new Matrix4f().perspective(Integer.MAX_VALUE, 1.3333334F, 9.0F, Integer.MAX_VALUE));
         RenderSystem.backupProjectionMatrix();
         RenderSystem.setProjectionMatrix(matrix4f);
 
 
         poseStack.pushPose();
         PoseStack.Pose pose = poseStack.last();
-        pose.pose().setIdentity();
-        pose.normal().setIdentity();
-        poseStack.translate(-3, -1.0, 1984.0);
-        poseStack.scale(4.5F, 4.5F, 4.5F);
-        poseStack.mulPose(Vector3f.YP.rotationDegrees(minecraft.level.getGameTime()));
-        poseStack.mulPose(Vector3f.XP.rotationDegrees(180.0F));
+        pose.pose().identity();
+        pose.normal().identity();
+        poseStack.translate(2, 0.25, 1984.0);
+        poseStack.scale(3F, 3F, 3F);
```

Comment:
Reason for scale changes?

Reply:
If you went to the Shell selection screen after the port, it was scaled, translated, and flipped upside down, I was accommodating that. Feel free to change it to look better :D